tests/node/rpc/structs/RoofStatusTests.js

```diff
@@ -0,0 +1,43 @@
+const SDL = require('../../../config.js').node;
+const RoofStatus = SDL.rpc.structs.RoofStatus;
+
+const Test = require('./../../../Test.js');
+const Validator = require('./../../../Validator.js');
+const BaseStructTests = require('./BaseStructTests');
+
+describe('RoofStatusTests', function () {
+    before(function () {
+        this.create = function () {
+            return new RoofStatus()
+                .setLocation(Test.GENERAL_GRID)
+                .setStatus(Test.GENERAL_DOORSTATUSTYPE)
+                .setState(Test.GENERAL_WINDOW_STATE);
+        };
+
+        this.getExpectedParameters = function (sdlVersion) {
+            return {
+                [RoofStatus.KEY_LOCATION]: Test.JSON_GRID,
+                [RoofStatus.KEY_STATUS]: Test.GENERAL_DOORSTATUSTYPE,
+                [RoofStatus.KEY_STATE]: Test.JSON_WINDOW_STATE,
```

Comment:
@ymalovanyi On line 21 does `Test.JSON_WINDOW_STATE` need to be changed to `Test.GENERAL_WINDOW_STATE`?

Reply:
No, because WindowState is a struct.